pkg/utils/utils.go

```diff
@@ -75,27 +75,36 @@ func BeautifyMD(data []byte) (string, error) {
 	return out, nil
 }
 
-// ReadREADME: returns the byte array of README.md of a project
-func ReadREADME(dbname, projectName string) ([]byte, error) {
+// GetProjectPath: returns the path to the README inside the project
+func GetProjectPath(dbname, projectName string) (string, error) {
 	path, err := db.GetRecord(dbname, projectName, c.ProjectPaths)
 	if err != nil {
 		actualName, err := db.GetRecord(dbname, projectName, c.ProjectAliasBucket)
 		if err != nil {
 			log.Printf("project: %v not a valid project\n", projectName)
-			return nil, errors.Join(ErrGetAlias, err)
+			return "", errors.Join(ErrGetAlias, err)
 		}
 		projectName = actualName
 		path, err = db.GetRecord(dbname, projectName, c.ProjectPaths)
 		if err != nil {
 			log.Printf("project: %v not a valid project\n", projectName)
-			return nil, errors.Join(ErrGetProject, err)
+			return "", errors.Join(ErrGetProject, err)
 		}
 	}
 	pPath := filepath.Join(path, "README.md")
```

Comment:
The code was already there, but I would like you to suggest trying multiple names (yes, it would need a refactoring)

Some projects have:
- README.md
- Readme.md
- readme.md
- ReadMe.md
- … all the above without a .md

Reply:
oh yeah totally , this makes sense. I will add this.

Reply:
Fixed with 

- #63